.gitignore

```diff
@@ -9,3 +9,4 @@ vendor/
 /phpunit.xml
 /.phpcs-cache
 /phpstan.neon
+/psalm.xml.dist
```

Comment:
Why exclude Psalm configuration from version control?

Reply:
mb, should have been no dist